docs/troubleshooting/reindex-elasticsearch.md

```diff
@@ -0,0 +1,29 @@
+---
+id: reindex-elasticsearch
+title: Reindexing Elasticsearch Guide
+hide_title: false
+---
+
+You may find yourself in a situation where you need to reindex your Elasticsearch index. If you have issues with your
+current index, or the mappings for an index have been updated (they do not update automatically), there are two ways
+to get the data synced correctly.
+
+## Change Index and Reindex
+
+The first way involves reindexing from the existing Elasticsearch index to a brand new index. This is done in two steps.
```

Comment:
We need to mention that for a period of time, while the reindex takes place, you will not see all of your data on the dashboard as it being brought over from the previous index. We should mention both options here. 1. you use the new index and the pros are you don't lose data, but the cons are it messes with your dashboard. 2. you reindex, then switch and the pros are your dashboard looks good, cons are you should stop ingest.